src/i18n/locale-en.json

```diff
@@ -219,6 +219,7 @@
     "ttyg": {
         "helpInfo": "Talk to Your Graph is chat bot based on OpenAI's GPT-4 model that can answer questions about data stored in GraphDB, leveraging both your data and the general knowledge of the GPT-4 model. The bot works in connection with the ChatGPT Retrieval connector.",
         "hint": "Talk to your graph – simply ask a question!",
```

Comment:
The translations also need to be added to "/test-cypress/fixtures/locale-en.json".